src/components/commons/Comment/Comment.tsx

```
@@ -1,158 +1,202 @@
-          {!isEditing ?
+          {!isEditing ? (
             <>
               <div css={profileContainer}>
                 <img css={profileImageStyle} src={roleImages[role]} />
                 <div css={profileTextContainer}>
                   <div css={nicknameContainer}>
-                    <a css={nicknameStyle(isMyComment)} onClick={handleNicknameClick}>{nickname}</a>
-                    {isWriter && <Badge content="작성자" color="blue" />}
+                    <a
+                      css={nicknameStyle(isCommentWriter)}
+                      onClick={handleNicknameClick}
```

Biome (1.9.4)

[error] Use a button element instead of an a element.

Anchor elements should only be used for default sections or page navigation
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)